src/markdown/tutorial/part-2/11-ember-data.md

```diff
@@ -152,12 +151,17 @@ The generator created some boilerplate code for us, which serves as a pretty goo
 
 This model test is also known as a *[unit test](../../../testing/testing-models/)*. Unlike any of the other tests that we've written thus far, this test doesn't actually *render* anything. It just instantiates the rental model object and tests the model object directly, manipulating its attributes and asserting their value.
 
-It is worth pointing out that EmberData provides a `store` *[service](../../../services/)*, also known as the EmberData store. In our test, we used the `this.owner.lookup('service:store')` API to get access to the EmberData store. The store provides a `createRecord` method to instantiate our model object for us.
+It is worth pointing out that EmberData provides a `store` *[service](../../../services/)*, also known as the EmberData store. In our test, we used the `this.owner.lookup('service:store')` API to get access to the EmberData store. The store provides a `createRecord` method to instantiate our model object for us. To make this `store` service available, we must add the following file:
+
```

Comment:
In ember-data > 5.3 beta, not having the file at app/services/store.js throws a deprecation

Reply:
I won't say I dislike that, having the store service imported from an explicit location is great, but I think this is an unexpected artifact. Will investigate.

Reply:
oh wait, no, I did :D 

The reason I also deprecated the app re-export is TS support. If we don't, too many things go wrong if you want to extend and use a store service in your app that is typed.

Reply:
👍 I commented on an ember-data issue that the deprecation needs a deprecation guide + url.